assets/onboarding/data/normalizer.js

```diff
@@ -0,0 +1,47 @@
+import { __ } from '@wordpress/i18n';
+
+import { INSTALLED_STATUS } from '../features/feature-status';
+
+/**
+ * Add details to title.
+ *
+ * @param {string}        title    Feature title.
+ * @param {string|number} price    Feature price.
+ * @param {string}        [status] Feature status.
+ */
+const addDetailsToTitle = ( title, price, status ) => {
```

Comment:
Just a minor improvement idea, this could be called like `addDetailsToTitle( feature )` with a signature of ` ( { title, price, status } ) => { ... }`.

WDYT about naming it something like `getTitleWithDetails` - `add` feels like it would modify something.

Reply:
Yeah! It's better! Refactored here: https://github.com/Automattic/sensei/pull/3176/commits/dbb97711700668e23902e58caf454c069856caef